src/main/java/javax/validation/Configuration.java

```diff
@@ -254,6 +269,17 @@
 	ClockProvider getDefaultClockProvider();
 
 	/**
+	 * Returns the default {@link ValueExtractor} implementations as per the
+	 * specification.
+	 *
+	 * @return the default {@code ValueExtractor} implementations compliant
+	 * with the specification
+	 *
+	 * @since 2.0
+	 */
+	Set<ValueExtractor<?>> getDefaultValueExtractors();
```

Comment:
Good point. I agree that this method may be not so useful if finding a specific extractor isn't straight forward. However, perhaps even the consistency argument is strong enough to keep it?